caster-front/tsconfig.vitest.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.app.json",
   "exclude": [],
   "compilerOptions": {
+    "moduleResolution": "node",
```

Comment:
Citing https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#moduleresolution-bundler

> If you are using a modern bundler like Vite, esbuild, swc, Webpack, Parcel, and others that implement a hybrid lookup strategy, the new bundler option should be a good fit for you.

As we are using Vite we should maybe use `bundler`? :) But I don't understand too much of it I must admit.

Reply:
Sorry, just readt the referenced issue - this seems a bug in ElementPlus, so maybe block the upgrade to TS5 until they have EP fixed their bug for TS5? Or is it necessary to upgrade to TS5? Switching to `node` resolution seems like a hack to me if TS advises to use `bundler`, especially since `node` seems somehow deprecated in favour of `nodenext`?

Reply:
Let's put it on hold then